### DISCUSSION TITLE
2024 4Q <성장 수집가> 김민기 OKR

### DISCUSSION THREAD

**Discussion by rlaalsrl0922** [Pseudo-Lab/growth-collector#19] [우리들의 성장 목표!]
O : 취업 뿌시기

KR1 : JD가 맞는 인턴십/채용 공고 "정성을 넣어" 지원하기, 석사관련 준비하기
KR2 : 최신 논문 요약 및 정리 블로그 운영하기
KR3 : AI 트랜드 꾸준히 팔로업 하기

**Comment by soncy0311** [comment]
화이팅입니다!!

**Comment by rlaalsrl0922** [comment]
**Action Item 2024.11.29 - 2024.12.5**

1. 공고 지원서 3개 이상 넣기
2. Gen AI 논문 stream 리스트업 하기
3. KSC 포스터 만들기

**Reply by yunhee1** [reply]
민기님 응원합니다~!

**Reply by SunandBean** [reply]
멋지십니다!!
